modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesKernelBoot.java

```diff
@@ -470,9 +470,11 @@ public static void stopHouseKeepingThreads() {
         log.info("Stop syncing exchanges, queues, bindings and subscriptions...");
         int threadTerminationTimePerod = 20; // seconds
         try {
-            andesRecoveryTaskScheduler.shutdown();
-            andesRecoveryTaskScheduler
-                    .awaitTermination(threadTerminationTimePerod, TimeUnit.SECONDS);
+            if(andesRecoveryTaskScheduler != null){
```

Comment:
At this point, these are not initialized. So, it gives the NullPointerException. Since it is OK to give that exception when shutting down, I'll remove these null checks.